src/quo2/components/wallet/account_card/style.cljs

```diff
@@ -102,3 +111,26 @@
 
 (def metrics-icon-container
   {:margin-left 4})
+
+(defn gradient-start-color
+  [theme customization-color]
+  (colors/theme-colors
+   (colors/custom-color customization-color 50 0)
+   colors/white-opa-2
+   theme))
+
+(defn gradient-end-color
+  [customization-color]
+  (colors/custom-color customization-color 50 6))
+
+(def gradient-view
+  {:position      :absolute
+   :bottom        0
+   :top           0
+   :left          0
+   :right         0
+   :border-radius 16})
+
+(defn alert-icon-color
+  [theme]
+  (colors/theme-colors colors/neutral-50 colors/white theme))
```

Comment:
No big deal @mmilad75, but there's a recommendation in the guidelines to not define component properties in the `style` ns. https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#dont-define-properties-in-styles-ns

These functions like `alert-icon-color` or `gradient-end-color` that are not returning maps are being used for component properties. One could argue the color is also a style, but another one could argue that the color is pre-processed in the icon component before being used, so it's not used "directly" as a style prop.

When we want to clean up the `view` namespace from component properties, we have been moving them to a `<component>/properties.cljs` file.

Reply:
Awesome!